geoserver_publisher/index.js

```diff
@@ -121,6 +124,40 @@ async function checkIfCoverageStoresExist(rasterMetaInf) {
   }
 }
 
+/**
+ * Creates a time-enabled layer in the GeoServer for the given raster mosaic if
+ * not existing by GeoServer REST-API.
+ *
+ * @param {Object} rasterMetaInf
+ */
+async function createRasterTimeLayers (rasterMetaInf) {
+  const ws = rasterMetaInf.workspace;
+  const covStore = rasterMetaInf.coverage_store;
+  const srs = 'EPSG:3035'; //TODO check if defined in DB
+  // per convention coverage store name is layer name and layer title
+  const layerName = covStore;
+  const nativeName = rasterMetaInf.properties_path;
+  const layerTitle = covStore;
+
+  verboseLogging(`Checking existence for layer ${ws}:${layerName} in coverage store ${covStore} (native name: ${nativeName})`);
+
+  const layer = await grc.layers.get(covStore);
+
+  if (!layer) {
+    console.info(`Creating layer "${ws}:${layerName}" in store "${covStore}"`);
+    // publishDbRaster (workspace, coverageStore, nativeName, name, title, srs, enabled)
```

Comment:
maybe a quick comment that this should describe the input of the function below. 
On first sight, I thought this was a commented functionCall